src/app/icd/ICDClientInfoManagement.cpp

```diff
@@ -0,0 +1,386 @@
+/*
+ *    Copyright (c) 2023 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
```

Comment:
can we please remove all `@file` comments in any PRs? They do not seem to provide value and sometimes they are copied & pasted and just lie (in this case it says this implements ICDClientPersistentStorage ... it does not, it implements ICDClientInfoManagement)

Reply:
done, thanks